src/test/resources/js/web.tests.js

```diff
@@ -99,3 +99,34 @@ describe("sprat.web.hateoas.embedded", function () {
         expect($hateoas.embedded(object)).toEqual(object._embedded.customers);
     });
 });
+
+describe("sprat.web.hateoas.constraints", function () {
+
+    it("the constraint 'delete' can be found", function () {
+        var object = {
+            constraints: {
+                delete: [
+                    {
+                        id: 111
+                    }
+                ]
+            }
+        };
+
+        expect($hateoas.constraints(object, 'delete')[0]['id']).toEqual(111);
+    });
+
```

Comment:
There should also be a test, checking for the expection in case there is no _constraints or .constraints object present.